app/src/main/java/net/gsantner/opoc/ui/SearchOrCustomTextDialog.java

```diff
@@ -328,7 +323,16 @@ public static void showMultiChoiceDialogWithSearchFilterUI(final Activity activi
         if ((w = dialog.getWindow()) != null && dopt.isSearchEnabled) {
             w.setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_ADJUST_RESIZE | WindowManager.LayoutParams.SOFT_INPUT_STATE_ALWAYS_VISIBLE);
         }
+
         dialog.show();
+
+        final Button neutralButton = dialog.getButton(AlertDialog.BUTTON_NEUTRAL);
+        if (neutralButton != null && dopt.neutralButtonText != 0 && dopt.neutralButtonCallback != null) {
+            neutralButton.setVisibility(Button.VISIBLE);
+            neutralButton.setText(dopt.neutralButtonText);
+            neutralButton.setOnClickListener((button) -> dopt.neutralButtonCallback.callback(dialog));
```

Comment:
Neutral button callback takes the _dialog_ as a parameter. Allows any logic to be moved to the callback instead of in the dialog class